timy/__init__.py

```diff
@@ -25,36 +25,52 @@ def output(ident, text):
 
 class Timer(object):
 
-    def __init__(self, ident=timy_config.DEFAULT_IDENT):
+    def __init__(self, ident=timy_config.DEFAULT_IDENT, include_sleep=True):
         self.ident = ident
         self.start = 0
+        self.end = 0
+        self.include_sleep = include_sleep
```

Comment:
What about holding the function in a variable and then calling it instead of using the if later?
Like: `self.time = time.perf_counter if include_sleep else time.process_time`